snuba/datasets/cdc/groupedmessage_processor.py

```diff
@@ -39,7 +59,9 @@ def from_wal(cls,
                 status=raw_data['status'],
                 last_seen=parse_postgres_datetime(raw_data['last_seen']),
                 first_seen=parse_postgres_datetime(raw_data['first_seen']),
-                active_at=parse_postgres_datetime(raw_data['active_at']),
+                active_at=parse_postgres_datetime(raw_data['active_at'])
+                if raw_data['active_at']
+                else None,
```

Comment:
Why did this change?

Also, this hard to read because of the indentation level — maybe put it between parentheses and indent it? (Same goes for line 81-83 below.)

Reply:
this change is because it was just broken before. active_at is nullable and parse_postgres_datetime does not deal with None.
I tried to indent it but black did not agree. I did not try to add parentheses though.